### DISCUSSION TITLE
How to download the roman ags bicolor font

### DISCUSSION THREAD

**Discussion by trewayy** [inkstitch/inkstitch#1747] [Q&A] [answered]
Hello, I'm trying to find the download to the roman ags bi-color font so I can import it into my website's personalization software. So far I could only find the license download on the inkstitch website. Is it even possible to download it?

**Comment by Augusa** [comment, marked as answer]
Roman AGS bicolor exits only as embroidery font for Inkstitch. I created it from Latin Modern... font as said in the license. Many bicolor fonts exist but I think you will have to pay for using it.
